takeshape/penny
# Update shopify instructions

### Test Plan (Steps to test):

1. Import a fresh Penny project and set up the Shopify services using the updated instructions. A good starting point is `Get your Storefront API keys.`
1. Run queries against both services in the api explorer to verify that they work

Example storefront query:

```graphql
query {
  ShopifyStorefront_product {
    id
  }
}
```

Example admin query:

```graphql
query {
  Shopify_app {
    availableAccessScopes {
      description
      handle
    }
  }
}
```

### Checklist:
